### PR TITLE
Test more rubies

### PR DESCRIPTION
**Pull request by dividedmind**
Adds Ruby 2.5 and 2.6.
Also removes patch version pinning so we automatically
test on the most current MRI in each branch.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ pipeline {
   stages {
```
Comment (izgeri) [inline Jenkinsfile]: I'm wondering if it makes sense at this point to add parallelization for these tests. We use this in other projects, for example here: https://github.com/conjurdemos/kubernetes-conjur-demo/blob/master/Jenkinsfile#L13

Comment (dividedmind) [reply Jenkinsfile]: Excellent point, I'll add it, thanks!

Comment (dividedmind) [reply Jenkinsfile]: Unfortunately it's not that simple. I created an issue for this, see #231.

Comment (izgeri) [reply Jenkinsfile]: Ok. It's fine if that gets added later, but for now there is still a failing build on this one.

**Comment by dividedmind** [comment]
This is ready for review. #232 has been fixed by removing the *ci_reporter_cucumber* gem.

File: features/pubkeys/show.feature [hunk 2]
```diff
@@ -12,7 +12,3 @@ Feature: Show public keys for a user
   Scenario: After adding a key, the key is shown
     When I run `conjur pubkeys show alice`
     And the output should match /^ssh-rsa .* laptop$/
-
-  Scenario: Public keys can be listed using cURL, without authentication
```
Comment (izgeri) [inline features/pubkeys/show.feature]: Why was this test removed?

Comment (dividedmind) [reply features/pubkeys/show.feature]: https://github.com/cyberark/conjur-cli/pull/230/commits/672867b109f65f57fe73f55f4a4f840b4e461544

> Remove a feature that tests only the appliance
> 
> This feature doesn't test the CLI at all, but just the appliance.
> There are similar tests elsewhere already, where they belong.

Comment (dividedmind) [reply features/pubkeys/show.feature]: Also, it was the only test that used a `Transform` feature that's incompatible with Cucumber 3.

**Pull request merged**